grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -156,6 +165,52 @@ class IssueService {
         issue
     }
 
+    Issue createIssue(IssueType type, Issue issue) throws DomainException {
```

Comment:
These methods are basically duplicates of `addIssue` and `updateIssue`.

Reply:
The idea is to remove old duplicated code as we move forward in the react migration functionality,  we kept this temporally in order to maintain stability on the system with previous info. 

We still need to work on some methods using the new logic for new Projects and new Consent group creation, to finally clean up. The plan is to create a Jira story in order to accomplish this, if it's ok for you.

Reply:
That sounds like a good plan, thanks for the explanation! 👍

Reply:
Not required, but might be nice to add `@Deprecated` to the things we plan on removing so we can keep track better.